app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/btle/profiles/deviceinfo/DeviceInfo.java

```diff
@@ -29,6 +29,7 @@
     private String systemId;
     private String regulatoryCertificationDataList;
     private String pnpId;
+    private String address;
```

Comment:
I think this is not part of the spec, so it should be either avoided or marked at least.

Reply:
How should I mark it? I would have avoided it if I would have seen a cleaner way of passing down for/from what device the message is meant for/coming from.